CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
 ## [1.18.68]
-### Fixed
+### Added
+- Source factors can now be added to the embeddings instead of concatenated with `--source-factors-combine sum` (default: concat)
 - Fixed training crashes with `--learning-rate-decay-optimizer-states-reset initial` option.
```

Comment:
Looks like you beat me to the merge. This should get a version bump here, though, right?

Reply:
yeah, true. Can you do that?